openpathsampling/volume.py

```diff
@@ -419,7 +422,16 @@ def __init__(self, collectivevariable, lambda_min = 0.0, lambda_max = 1.0,
 
     def do_wrap(self, value):
         """Wraps `value` into the periodic domain."""
+
+        # this looks strange and mimics the modulo operation `%` while
+        # being folly compatible for simtk numbers and plain python as well
+        # ints and floats. I think we usually have floats, but the tests
+        # from dwhswenson use ints and it should work with both
```

Comment:
If you used degrees, you're likely to give `int`s as input. Of course, maybe the better way is to make sure they get cast to float at some point.